### ISSUE TITLE
Help needed in Arctos - problem inviting operator

### ISSUE THREAD

**Issue by keg34** [ArctosDB/arctos#8303]
# Tell us what you are trying to do 
I'm trying to invite an operator - Cassie Stitzman, username = cstitzma
I'm getting the error below.  She is set up as an agent with necessary info such as email address.  I'm not sure what is meant by "only one login is possible".  Please advise, and thanks!

# What are relevant pages in Arctos
**Provide a link to or a description of the page where you need help.**
![image](https://github.com/user-attachments/assets/081def73-5409-43bd-9f07-efa0b63a10a3)

**Comment by dustymc** [comment]
That's super-weird, user never accepted a very old invitation maybe??

Anyway, I just flushed out the stuck bits, you should be able to start the invitation process over, and please let me know if you have any trouble or know what might have lead to this situation.

**Comment by keg34** [comment]
Thanks - that seems to have done the trick.  Haven't got the faintest what caused it.  I don't think that I have ever invited this user as an operator before, though that is not impossible.  Except that I apparently had not linked her user account to her agent yet, so that would mean that an invitation had never been sent before, right? (unless that got flushed when you did whatever you did?)  I did notice that her email showed up in two places in her agent record (email and notification email).  When I was trying to figure out what was wrong I deleted the "notification email" thinking that maybe emails could only be tied to one address type, but that didn't solve the issue.  I wouldn't think that is the problem, but it's all that comes to mind.
